tests/pytest_loganalyzer/pytest_loganalyzer.py

```diff
@@ -0,0 +1,181 @@
+import sys
+import logging
+import os
+import re
+
+from time import gmtime, strftime
+from os.path import join, split
+from os.path import normpath
+
+# Integration with legacy loganalyzer
+ANSIBLE_LOGANALYZER = normpath(join(split(__file__)[0], "../../ansible/roles/test/files/tools/loganalyzer/loganalyzer.py"))
+sys.path.append(split(ANSIBLE_LOGANALYZER)[0])

```

Comment:
IMO, symbolic link to loganalyzer.py in same directory would be better

Reply:
Fixed, added symbolic link.